my-site.rb

```diff
@@ -0,0 +1,24 @@
+require "sinatra"
+require "sinatra/reloader"
+require './lib/scrabble'
+
+class MySite < Sinatra::Base
+  register Sinatra::Reloader
+
+  get "/" do
+    erb :home
+  end
+
+  get"/score" do
+    erb :score
+  end
+
+  post "/score" do
+    @word = params[:scrabble][:word]
+    params[:scrabble][:score] = Scrabble::Scrabble.score(@word)
```

Comment:
Does this need to be in the `params` object, or can the score just be set directly equal to `@score`?